plugins/mail/postfix_stats

```diff
@@ -3,50 +3,102 @@
 # Plugin to show Postfix statistics - needs pflogsumm
 #
 # Contributed by David Obando (david@cryptix.de) - 16.04.2007
+# Rewrited by Cristian Deluxe (me@cristiandeluxe.com) - 02.11.2016
+#
+#
+# Example config for Ubuntu (You need: apt-get install pflogsumm)
+#
+# [postfix_stats]
+# env.logfile /var/log/syslog
+# env.logfile2 /var/log/syslog.1
+# env.pflogsumm pflogsumm
 #
 #
 # Magic markers - optional - used by installation scripts and
 # munin-config:
 #
-#%# family=manual
+#%# family=contrib
 #%# capabilities=autoconf
 
 #set -xv
+SYS_LOG=${logfile:-/var/log/syslog}
+SYS_LOG2=${logfile2:-/var/log/syslog.0}
+PFLOGSUMM=${pflogsumm:-pflogsumm.pl}
+
+# Fields (Array to avoid code duplication)
+declare -a FIELDS_ARR=("received" "delivered" "forwarded" "deferred" "bounced" "rejected" "held" "discarded")
+
+#
+# Autoconf Section
+#
+if [ "$1" = 'autoconf' ]; then
+	# Try to find pflogsumm with default name
+	PFLOG_EXIST=$(command -v pflogsumm.pl)
+
+	# Try to find pflogsumm without any extension
+	if [[ -z "${PFLOG_EXIST}" ]]
+		then
+		PFLOG_EXIST=$(command -v pflogsumm)
+	fi
+
+	if [[ -z "${PFLOG_EXIST}" ]]
+		then
+		echo 'no';
```

Comment:
you can save one character here :)

Additionally you should add the reason for "no" in brackets afterwards. E.g.: `echo "no (failed to find pflogsum)"`